src/components/html-injection/ko/htmlInjectionViewModelBinder.ts

```diff
@@ -0,0 +1,88 @@
+import { Bag } from "@paperbits/common";
+import { EventManager, Events } from "@paperbits/common/events";
+import { ComponentFlow, IWidgetBinding } from "@paperbits/common/editing";
+import { widgetName, widgetDisplayName, widgetEditorSelector } from "../constants";
+import { HtmlInjectionViewModel } from "./htmlInjectionViewModel";
+import { ViewModelBinder } from "@paperbits/common/widgets";
+import { StyleCompiler } from "@paperbits/common/styles";
+import { JssCompiler } from "@paperbits/styles/jssCompiler";
+import { ISettingsProvider } from "@paperbits/common/configuration";
+import { HTMLInjectionModel } from "../htmlInjectionModel";
+
+/**
+ * This class describes how the model needs to be presented (as a view model)
```

Comment:
These comments left from tutorial:  `This class describes how the model needs to be presented (as a view model)...`